host/xtest/xtest_uuid_helpers.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2014, STMicroelectronics International N.V.
```

Comment:
Maybe ` * Copyright (c) 2019, Linaro Limited` as per new source file **xtest_uuid_helpers.c**.

Reply:
As this is a copy from same project other git tree I would not like to change it. But in here I can put what ever is recommended action from project team. Probably should be own commit after the copy or so.

Reply:
When copying an existing file I don't change the copyrights, I simply add mine if I modify the code.